### ISSUE TITLE
Ctrl + Tab order

### ISSUE THREAD

**Issue by okneloper** [HeidiSQL/HeidiSQL#344]
## Steps to reproduce this issue

1. Press Ctrl + T to open a new Query tab
2. Press Ctrl + Tab to switch tabs

## Current behavior
The tabs are switched in order of appearance on the tab panel.

## Expected behavior
It would be nice to have an option to list tabs in the order they were accessed previously. Would be a great productivity enhancement.

* **HeidiSQL version:** 
  Версия 9.5.0.5293 (64 Bit)

**Comment by endo64** [comment]
This is a personal taste, I usually prefer switching in order of appearance, the other way confuses me. Could be optional though.

**Comment by ansgarbecker** [comment]
Yes, this should really be optional if at all implemented.
I'm tending to say "this is just the way HeidiSQL does it", but if you say that this would enhance productivity for some people we'd leave this open. But the subject phrasing is horrible...

**Comment by okneloper** [comment]
I am surprised to see doubts this enhancing productivity :)
I mean how else do you select the tab you need out of umpteen open tabs with nearly identical titles?

**Comment by jonfre** [comment]
For me this is actually the only thing that makes Heidi hard to use. Navigating more than three tabs is a pain. Ctrl+Tab Most Recent Used behaviour is the standard in software for developers, including SQL Server Management Studio.

**Comment by kovacsbv** [comment]
Yes, flipping back and forth between a query and data in a table or between two queries is a very common use case.  If you run with 5-10 tabs open like I do, you have to watch the tabs, count them, or put a lot of effort into switching back and forth.  And, of course, the keyboard is a lot faster than the mouse.  The term I've heard for this is Most Recently Used (MRU).  As with many matters of preference, it should have a checkbox to select what you want to do.

**Comment by endo64** [comment]
The text editor I use (Editplus) has two distinct shortcuts for this behavior,
1. CTRL-Tab / CTRL-Shift-Tab for switching tabs forward and backward in the order of appearance.
2. F12 (configurable) to switch between the last two documents

Which I find it more useful than changing CTRL-Tab's default, as I said it is easier for me to switch tabs in the order of tabs (a personal taste, but I find it difficult to not easily switch to the next tab of the active tab because of the recent used one is a different tab, then I can't even count the tabs because it looks almost unordered)

So if it will ever implemented, please make it another shortcut key.

**Comment by okneloper** [comment]
There are arguably more people that will be used to the Ctrl+Tab combination for switching tabs than any other key combination.

**Comment by kovacsbv** [comment]
That's exactly right.  There's lots of arguing going on.  It's also arguable that MRU is more efficient and that more people want MRU.  Lots of people like it one way or the other.  Put a checkbox in options to select MRU.  Leave it off by default so the behavior is the same as it always was.  Let people who want MRU check the thing.  It's the most arguable that more people will be happy this way.
